src/controllers/usersController.js

```diff
@@ -114,12 +114,18 @@ const deleteUser = async (req, res, next) => {
 const loginUser = async (req, res, next) => {
 
   // input validation
-  var attrs = ["email", "password"];
-  var contype = req.headers['content-type'];
-  if (!contype || contype.indexOf('application/json') !== 0) res.sendStatus(415);
-  else if (!inputValidation.hasAllAttrs(req.body, attrs)) res.status(400).send("Please provide required input attributes");
-  else if (inputValidation.includesNullorEmpty(req.body)) res.status(400).send("Please provide required input values");
-  else {
+  let success = true;
+  try {
+    //check content type
+    if(!inputValidation.checkContentType(req, res)) throw new ContentTypeError();
+    //check null values
+    errList = inputValidation.getNullorEmpty(res, req.body);
```

Comment:
Are we missing error throwing here? The errList is not used

Reply:
Updated, thanks!